Source/JavaScriptCore/dfg/DFGLICMPhase.cpp

```diff
@@ -63,6 +63,8 @@ class LICMPhase : public Phase {
     
     bool run()
     {
+        m_graph.m_planStage = PlanStage::LICMAndLater;
```

Comment:
Let's insert assertion which is `ASSERT(m_graph.m_planStage <= PlanStage::LICMAndLater);` before setting to ensure that we are not decrementing the phase.